extension/scripts/features/item-e-warning/ttItemEWarning.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+(async () => {
+	if (!getPageStatus().access) return;
+
+	const feature = featureManager.registerFeature(
+		"Item Over E Warning",
+		"items",
+		() => settings.pages.items.itemEWarning,
+		initialiseListener,
+		addWarning,
```

Comment:
Should be undefined, as it will be returned immediately anyway (since `item` will be falsely).